src/test/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/evaluering/EvalueringMålgruppeTest.kt

```diff
@@ -0,0 +1,116 @@
+package no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.evaluering
+
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.DelvilkårMålgruppe
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.DelvilkårVilkårperiode
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.MålgruppeType
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.ResultatDelvilkårperiode
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.ResultatVilkårperiode
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.SvarJaNei
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.dto.DelvilkårMålgruppeDto
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.evaluering.EvalueringMålgruppe.utledResultat
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.EnumSource
+
+class EvalueringMålgruppeTest {
+
+    val svarJa = DelvilkårMålgruppeDto(SvarJaNei.JA)
+    val svarImplisitt = DelvilkårMålgruppeDto(SvarJaNei.JA_IMPLISITT)
+    val svarNei = DelvilkårMålgruppeDto(SvarJaNei.NEI)
+    val svarMangler = DelvilkårMålgruppeDto(null)
+
+    @Nested
+    inner class Implicitt {
+        @ImplisittParameterizedTest
+        fun `mangler svar skal mappes til oppfylt`(type: MålgruppeType) {
+            val resultat = utledResultat(type, svarMangler)
```

Comment:
Skal man få lov å ha manglende svar på medlemskap/målgruppe?

Reply:
Mulig ikke-aktuellt og ikke_vurdert som svar er tydligere

Reply:
Etter diskusjon valgte vi å gå for null